packages/forma-36-codemod/README.md

```diff
@@ -0,0 +1,33 @@
+## forma-36-codemod
```

Comment:
I would suggest going with the shorter `f36-codemod` naming convention we're changing to,  here and wherever else it's used.

Reply:
+1

Reply:
Should this also be applied to the package name, or only on the readme?

Reply:
package name as well